pages/common/atom.md

```diff
@@ -0,0 +1,11 @@
+# atom
+
+> A hackable text editor for the 21st century.
```

Comment:
This is not a place for their motto. It is a place for description of command.
So it should be rather 
```md
> Atom text editor
```